float64 mixing_factor
geometry_msgs/Point[] mean